describealign.py

```diff
@@ -1187,20 +1187,11 @@ def main_gui():
 # Entry point for command line interaction, for example:
 # > describealign video.mp4 audio_desc.mp3
 def command_line_interface():
-  # override command line argument parser's error handler to make it pause before exiting
-  # this allows users to see the error message when accidentally not running from command line
```

Comment:
Why are we removing this?  I like this feature

Reply:
It's not being removed, just changed so that if any args are specified it won't open the gui. Currently if you try `describealign --stretch_audio` for instance, instead of printing the command usage it opens the GUI. With the new check `if len(sys.argv) < 2:` it will only open the GUI if the app is run with no args